manifests/init.pp

```diff
@@ -64,6 +64,9 @@
 
   # Padoptions
   Etherpad::Padoptions $padoptions = {},
+
+  # Plugins
+  Hash[Pattern['ep_*'], Variant[Boolean, Undef]] $plugins_list = {},
```

Comment:
`Variant[Boolean, Undef]` can also be written as `Optional[Boolean]`

Reply:
Fixed.